src/Components/ProvisioningWizard/ProvisioningWizard.test.js

```diff
@@ -36,13 +36,9 @@ describe('ProvisioningWizard', () => {
 
       render(<ProvisioningWizard hasAccess image={{ ...awsImage, sourceIDs: ['1'] }} />);
       // wait for the sources to load
-      await screen.findByText('Select account', undefined, { timeout: 10000 });
```

Comment:
this is redundant check, just make the test be slower, if the `Source 1` exists, we don't need the `Select account` placeholder.